dev/com.ibm.ws.jaxrs.2.1.client_fat/fat/src/com/ibm/ws/jaxrs21/client/fat/test/JAXRS21ClientSSLProxyAuthTest.java

```diff
@@ -474,15 +447,13 @@ public void testTunnelThroughProxyToHTTPEndpointInvalidUsername_Builder() throws
         assertEquals("Proxy Password value printed in trace output", 0, server.findStringsInLogsAndTraceUsingMark("myPa\\$\\$word").size());
         server.setMarkToEndOfLog(server.getFileFromLibertyServerRoot("logs/trace.log"));
 
-        // TODO Enable after MockServer fix has been delivered
-        // proxyClient.verify(request().withHeader("Proxy-Authorization", "Basic
-        // amF4cnNVc2VyMTpteVBhJCR3b3Jk").withSecure(false)); //jaxrsUser1:myPa$$word
+        proxyClient.verify(request().withHeader("Proxy-Authorization", "Basic amF4cnNVc2VyMTpteVBhJCR3b3Jk").withSecure(false)); //jaxrsUser1:myPa$$word
     }
 
-    // @Test // TODO: Enable after fixing Open-Liberty issue 227
+//    @Test // TODO: Enable after fixing Open-Liberty issue 227
```

Comment:
Looks like issue #227 is fixed.  Can you try re-enabling this test to see if it passes now?

Reply:
I looked into that but wanted to keep it separate from this pull request, particularly since I'm having trouble running the entire bucket locally.